packages/jest-core/src/getProjectDisplayName.ts

```diff
@@ -10,9 +10,5 @@ import type {Config} from '@jest/types';
 export default function getProjectDisplayName(
   projectConfig: Config.ProjectConfig,
 ): string | undefined {
-  const {displayName} = projectConfig;
-  if (!displayName) {
-    return undefined;
-  }
-  return displayName.name;
+  return projectConfig?.displayName?.name ?? undefined;
```

Comment:
Are you sure that it should be `??` instead of `||`?

Reply:
Could the user name the project as `''` (empty string)? If they can, then it should be `||` (or), otherwise it is correct like that (I guess)...

Reply:
The user _could_, yeah. I guess returning `undefined` is better than an empty string in that case? Not just because it matches the current behaviour, but also because (imo) it seems more natural?

Reply:
Yeah, (imo) doesn't make to much sense allows empty strings but I will change because I can see problems happening if instead of `undefined` the return were an empty string